src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.text1 {
```

Comment:
the name of the selector should be more descriptive. Looks like this one is not actually a text, but a searchbar, no?

Reply:
you're right i will fix this too